lstchain/tools/lstchain_create_irf_files.py

```diff
@@ -90,9 +106,19 @@ class IRFFITSWriter(Tool):
         -g /path/to/DL2_MC_gamma_file.h5
         -o /path/to/irf.fits.gz
         --point-like (Only for point_like IRFs)
-        --fixed-gh-cut 0.9
-        --fixed-theta-cut 0.2
+        --global-gh-cut 0.9
+        --global-theta-cut 0.2
         --irf-obs-time 50
+
+    Or use energy-dependent cuts based on a gamma efficiency:
+    > lstchain_create_irf_files
+        -g /path/to/DL2_MC_gamma_file.h5
+        -o /path/to/irf.fits.gz
+        --point-like (Only for point_like IRFs)
+        --energy-dependent-gh
+        --energy-dependent-theta
+        --gh-efficiency 95
+        --theta-containment 68
```

Comment:
these parameters should be 0.95 and 0.68 now?

Reply:
True! Lemme check elsewhere as well